src/Flex.jsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import cx from 'classnames';
+
+
+export const VALID_ALIGNMENTS = {
+	top: 'Top',
+	bottom: 'Bottom',
+	center: 'Center',
+};
+
+export const VALID_BREAKPOINTS = {
+	all: 'atAll',
+	medium: 'atMedium',
+	large: 'atLarge',
+};
+
+export const VALID_SPACE = {
+	center: 'center',
+	around: 'spaceAround',
+	between: 'spaceBetween',
+	flex: 'flexEnd'
+};
+
+export const DIRECTION_ROW = 'row';
+export const DIRECTION_COLUMN = 'column';
+
+export const FLEX_CLASS = 'flex';
+export const FLEX_ROW_CLASS = `${FLEX_CLASS}--${DIRECTION_ROW}`;
+export const FLEX_COLUMN_CLASS = `${FLEX_CLASS}--${DIRECTION_COLUMN}`;
+export const FLEX_WRAP_CLASS = `${FLEX_CLASS}--wrap`;
+export const FLEX_NOGUTTER_CLASS = `${FLEX_CLASS}--noGutters`;
+export const FLEX_ALIGN_CLASS = `${FLEX_CLASS}--align`;
+
+/**
+ * Design System Component: Provides `Flex` styled container for ideal use with `FlexItem` content
+ * @module Flex
+ */
+class Flex extends React.Component {
+	/**
+	 * @return {React.element} the commend form React element
+	 */
+	render() {
+		const {
+			direction,
+			switchDirection,
+			wrap,
+			noGutters,
+			justify,
+			align,
+			rowReverse,
+			columnReverse,
+			children,
+			className,
+			...other
+		} = this.props;
+
+		const columnReverseBreakpoint = VALID_BREAKPOINTS[columnReverse] || VALID_BREAKPOINTS['all'];
+		const rowReverseBreakpoint = VALID_BREAKPOINTS[rowReverse] || VALID_BREAKPOINTS['all'];
+
+		const isColumn = direction === DIRECTION_COLUMN;
+		const classNames = cx(
+			FLEX_CLASS,
+			{
+				// horizontal default
+				[FLEX_ROW_CLASS]: !isColumn,
```

Comment:
I know the square brackets are not needed for plain strings like `FLEX_ROW_CLASS`. Is this just a style thing to keep it consistent with the interpolated strings? I can appreciate that.